lib/cli.js

```diff
@@ -78,10 +78,23 @@ function init() {
     // start the interactive UI if no generator is passed
     if (!cmd) {
       if (opts.help) {
-        var genList = Object.keys(env.getGeneratorsMeta()).map(function (el) {
-          var parts = el.split(':');
-          var first = parts.shift();
-          return '  ' + (parts[0] === 'app' ? first : '  ' + parts.join(':'));
+        var generators = Object.keys(env.getGeneratorsMeta()).reduce(function (acc, gen) {
+          var parts = gen.split(':');
+          var genName = parts.shift();
+          // If first time we found this generator, prepare to save all its sub-generators
+          if (!acc[genName]) {
+            acc[genName] = [];
+          }
+          // If sub-generator (!== app), save it
+          if (parts[0] !== 'app') {
+            acc[genName].push(parts.join(':'));
+          }
+
+          return acc;
+        }, {});
+
+        var genList = Object.keys(generators).map(function (gen) {
+          return '  ' + gen + '\n' + generators[gen].map(function (sub) { return '    '+sub; }).join('\n');
```

Comment:
Move this on multiple line.

Also, always add spaces between operators (in this case before and after `+`).